doc/examples.xml

```diff
@@ -703,3 +703,45 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="BookDigraph">
+<ManSection>
+  <Oper Name="BookDigraph" Arg="[filt, ]m"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    The <E>Book digraph</E> is (generally) the cartesian product of the chain digraph
+    (as the "Book Spine") and the star digraph (as the "Pages"). When <A>m</A> is less than 3
+    two special cases occur. When <A>m</A> is 1, a (<A>2</A>, <A>2</A>) complete bipartite
+    graph is returned (the "square" graph). When <A>m</A> is 2, a (<A>2</A>, <A>3</A>)
+    square grid graph is returned (the "domino" graph). For more details on the Book digraph
+    please refer to <URL>https://mathworld.wolfram.com/BookGraph.html</URL>.<P/>
+
+    See <Ref Oper="DigraphCartesianProduct"/>,
```

Comment:
As can be seen on the GitHub Actions artifact (containing the compiled documentation), this reference doesn't resolve (note the `???`).

Screenshot of the documentation:

<img width="1171" alt="Screenshot 2021-03-18 at 21 00 10" src="https://user-images.githubusercontent.com/8956868/111697151-19a3c080-882d-11eb-9b08-c98322d8ec27.png">
This is probably because there are actually multiple bits of documentation with the name `DigraphCartesianProduct`, and so you need to specify which one you're referencing.

## Fix

To work out how to properly reference `DigraphCartesianProduct`, we should search the Digraphs package's `doc/` folder for the place where `DigraphCartesianProduct` is documented. We then learn that is is documented in `doc/oper.xml`, as follows:
```
<#GAPDoc Label="DigraphCartesianProduct">
<ManSection>
  <Func Name="DigraphCartesianProduct" Arg="gr1, gr2, ..."
    Label="for a positive number of digraphs"/>
  <Func Name="DigraphCartesianProduct" Arg="list"
    Label="for a list of digraphs"/>
  <Returns>A digraph.</Returns>
  <Description>
```
... and so on.

So in fact it is a `Func` rather than an `Oper`, and there are two different labels that we can reference – since the two versions of `DigraphCartesianProduct` are documented together, it doesn't matter which one we reference, since any link will take us to this same bit of documentation. So let's just pick the first.

Therefore the fix is:
```suggestion
    See <Ref Func="DigraphCartesianProduct" Label="for a positive number of digraphs"/>,
```

Reply:
I see, thank you! Is there a way to compile the documentation locally (just curious)?

Reply:
Yes, there's a couple of ways 🙂. Within Digraphs, you can run `DigraphsMakeDoc();`. Or, within GAP, you can read in the `makedoc.g` file of the Digraphs package via the `Read` command. 

This produces the HTML and PDF versions of the manual. So in particular I think it'll fail if you don't have `pdflatex` installed.

Reply:
When you compile the manual (e.g. locally), there is also a lot of output, and that output contains warnings about broken references like this one.

Reply:
Thank you for this!